src/components/pill-selector-dropdown/__tests__/PillSelector.test.js

```diff
@@ -41,6 +41,13 @@ describe('components/pill-selector-dropdown/PillSelector', () => {
             expect(wrapper).toMatchSnapshot();
         });
 
+        test('should render with disabled input', () => {
+            const wrapper = shallow(<PillSelector inputDisabled onInput={() => {}} onRemove={() => {}} />);
+            const input = wrapper.find('textarea');
+
+            expect(input.prop('disabled')).toBeTruthy();
```

Comment:
we can be more explicit
```suggestion
            expect(input.prop('disabled')).toBe(true);
```